lib/jekyll/static_file.rb

```diff
@@ -19,7 +17,10 @@ def initialize(site, base, dir, name, collection = nil)
       @collection = collection
       @relative_path = File.join(*[@dir, @name].compact)
       @extname = File.extname(@name)
+      # The cache of last modification times [path] -> mtime.
+      @mtimes = {}
```

Comment:
It was this error:
```
lib/jekyll/static_file.rb:4:5: C: Style/ClassVars: Replace class var @@mtimes with a class instance var.
    @@mtimes = {}
    ^^^^^^^^
```

What is the alternative way? Sorry, I'm a newbie in Ruby 😄